sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/QuickFilters.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from 'react';
+import { SelectorGroup, Selector } from '../../../../components';
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusUnknownSmall,
+  StatusPlaceholderSmall,
+} from 'grommet-icons';
+import { DataContext, Text } from 'grommet';
+
+interface DataContextType {
+  onView: (view: any) => void;
+  view: any;
+}
+
+export const QuickFilters: React.FC<{
+  value: any;
+  setValue: any;
+  counts: {
+    down?: number;
+    up?: number;
+    unknown?: number;
+    undefinedState?: number;
+  };
+}> = ({ value: selectedValue, setValue, counts }) => {
+  const { onView, view } = useContext(DataContext) as DataContextType;
+
+  return (
+    <SelectorGroup
+      a11yTitle="Server State quick filters"
+      value={selectedValue}
+      defaultValue={undefined}
+      multiple={false}
+      onSelect={({ value }) => {
+        const nextView = {
+          ...view,
+          search: '',
+          page: 1,
+          properties: value ? { state: [value] } : {},
+          value: undefined,
+        };
+        onView(nextView);
+        setValue(value);
+      }}
+      layout="grid"
+    >
+      <Text size="large">Group By Availability State</Text>
```

Comment:
Lets remove this text. Based on the mocks, this is acting as a sort on cluster status. We should implement a <DataSort /> by cluster status.